libs/application/core/src/validation/validators.spec.ts

```diff
@@ -317,9 +322,7 @@ describe('validateAnswers', () => {
       } as FormValue
       const secondError = validateAnswers(schemaWithArray, anotherBadFormValue)
       expect(secondError).toEqual({
-        person: invalidInput,
-        'person[1]': invalidInput,
-        'person[1].age': invalidInput,
+        person: [undefined, { age: invalidInput }],
```

Comment:
Should we add tests for this changed functionality? Maybe especially some regarding arrays?

Reply:
I think this is sufficient for arrays with objects.